monai/handlers/checkpoint_loader.py

```diff
@@ -79,11 +84,36 @@ def __call__(self, engine: Engine) -> None:
         Args:
             engine: Ignite Engine, it can be a trainer, validator or evaluator.
         """
-        checkpoint = torch.load(self.load_path, map_location=self.map_location)
-        if len(self.load_dict) == 1:
-            key = list(self.load_dict.keys())[0]
-            if not (key in checkpoint):
-                checkpoint = {key: checkpoint}
+        if self.find_latest:
+            models = sorted(os.listdir(self.load_path))
```

Comment:
`glob.glob(self.load_path+"/*.pt")` would give you a list of full paths to .pt files.